### DISCUSSION TITLE
Parsing JSON

### DISCUSSION THREAD

**Discussion by pathavyer** [illacloud/illa-builder#536] [Q&A] [answered]
Just doing some test and I kept getting [object Object], .. when I try to parse JSON

**Comment by AruSeito** [comment, marked as answer]
thx for your feedback!  We will optimize it shortly.

**Reply by pathavyer** [reply]
Thanks @AruSeito! It was my mistake, I didn't see the transformer area. I was able to parse the objs that I was testing via transformer.

**Reply by AruSeito** [reply]
Hello,@pathavyer .It's great that you can solve this problem in other ways! Can Illa meet your need to simplify the workflow? Or do you think there are any deficiencies in illa?

**Reply by pathavyer** [reply]
@AruSeito , I've only scratched the surface of the application, I think it has really great potential and definitely a huge amount of work has been invested to this, kudos to you and the team. So far my only comment would be regarding documentation, I think it will continue to grow and for it to attract more users, solid documentation would be a must.

**Reply by AruSeito** [reply]
Thank you for your valuable comments! We will continue to optimize the document！
